jsapp/js/components/header/mainHeader.component.tsx

```diff
@@ -99,7 +71,10 @@ const MainHeader = class MainHeader extends Reflux.Component {
   }
 
   renderGitRevInfo() {
-    if (sessionStore.currentAccount?.git_rev) {
+    if (
+      'git_rev' in sessionStore.currentAccount &&
+      sessionStore.currentAccount?.git_rev
```

Comment:
Confused by this change. At least, if 'git_rev' is `in sessionStore.currentAccount`, then we shouldn't need `?.` here, right?

Reply:
Right, I've changed it to check if both of the values we use for rendering are strings.